Tiltfile

```diff
@@ -1,31 +1,115 @@
+def printup(*args):
+  if config.tilt_subcommand == "up":
+    print(*args)
+
+config.define_bool('local-kibana')
+
+parsed_config = config.parse()
+
+script_dir = os.path.join(config.main_dir, 'script')
+
+default_kibana_port = 5601
+kibana_host = "localhost"
+kibana_base_path = ""
+
+# checks whether (dev mode) kibana is already running at given port,
+# and returns the URL it is redirected to
+def discover_kibana():
+
+    default_kibana_host = "localhost"
+
+    count = int(
+      str(
+        local("lsof -n -i :%s | grep LISTEN | wc -l | tr -d ' '" % default_kibana_port, quiet=True, echo_off=True)
+      ).strip()
+    )
+
+    if count <= 0:
+      fail("No service found on port " + str(default_kibana_port))
+      return None
+
+    is_docker_kibana = str(
+      local(
+        """curl --fail -s -w '%header{kbn-name}'""" + " http://{}:{}".format(default_kibana_host, default_kibana_port),
+        quiet=True,
+        echo_off=True
+      )
+    ).strip() == "kibana"
+
+    if (is_docker_kibana):
+      return None
```

Comment:
Same as above -- can we just assume that it's running on the host if `--local-kibana` has been passed?